data-raw/data_mile1.R

```diff
@@ -15,6 +15,10 @@
 ###############################################################################
 # Helper functions and load packages
 ###############################################################################
+cv_2_sd <- function(x) {
+  log(sqrt(log(x^2 + 1)))
```

Comment:
I think I need help from @Andrea-Havron-NOAA here. The `cv_2_sd()` function is designed to convert the CVs from model comparison project for the `log_obs_error` in FIMS fleet module. To me, standard deviation on the log scale is calculated from the specified CVs using the relationship `sdlog=sqrt(log(CV^2+1))`. Does the additional `log` outside of `sqrt(log(CV^2+1))` imply that we log the standard deviation on the log scale? Do we need to consider renaming the function to `cv_2_logsd()`?

Reply:
The outside log is the parameter transformation where internally the model takes the exponential to keep the value positive. FIMS will be using internal transformations to bound parameters. Most parameters will need to be transformed so my personal preference would be to have the cv_2_sd return the value without the outer log and then have a separate function that applies the correct transformation based on the parameter.

Reply:
But these aren't parameters so I don't think it's necessary to log it internally? It'll just be exponentiated again. I'm not aware of any assessments that will estimate these in each year. At most an additional variance term is estimated on top of the supplied values.